.mypy.ini

```diff
@@ -0,0 +1,26 @@
+[mypy]
+files = tests/test_multidict.py, tests/test_mypy.py
```

Comment:
We can expand this to other files later. But, the previous mypy check was not checking anything anyway.